config/configgrpc/configgrpc.go

```diff
@@ -94,12 +94,12 @@ type GRPCClientSettings struct {
 	// The headers associated with gRPC requests.
 	Headers map[string]string `mapstructure:"headers"`
 
-	// PerRPCAuth parameter configures the client to send authentication data on a per-RPC basis.
```

Comment:
Food for thought: can't we keep this for a couple of releases, deprecating it first, and then completely removing it? I think it should be possible to build an Auth struct out of the information from the PerRPCAuth instance.

Reply:
Keeping the PR as is per @gramidt suggestion below. @jpkrohling please let me know, I can always open another PR with PerRPCAuth otherwise. I do think I think it is ok to break the compatibility for Auth as we move forward towards 1.0 and enforce backward compatibility afterwards.